arwen/elrondapi/managedei.go

```diff
@@ -708,6 +755,7 @@ func v1_4_managedExecuteReadOnly(
 		vmInput.arguments,
 	)
 	setReturnDataIfExists(host, lenReturnData, resultHandle)
+	runtime.TraceGasUsed(managedExecuteReadOnlyName, initialGasLeft-metering.GasLeft())
```

Comment:
we actually want to trace the API call - not necessarily the whole execution below it. You already have that cost in the underlying execution. Here you will have duplicated stuffs as gas. You will have the same gas consumed twice in gasTrace